config/locales/en.yml

```
@@ -148,3 +150,23 @@ en:
   max_hints_count: 'Number of available hints'
   loading_please_wait: 'Loading... Please wait...'
   choose_another_answer: 'Wrong, choose another answer.'
+  edit: 'Edit'
+  delete: 'Delete'
+  tags_name: 'Tags name'
+  actions: 'Actions'
+  add: 'Add'
+  remove: 'Remove'
+  available_tasks: "Available tasks"
+  create_tag: 'Create tag'
+  enter_tag_name: 'Enter name'
+  edit_tag: 'Edit tag'
+  filter_by_tags: 'Filter by tags'
+  reset: 'Reset'
+  edit_users: 'Edit users'
+  users_list: 'List of users'
+  email: 'Email'
+  role: 'Role'
+  created_at: 'Created at'
+  confirmation_deletion_user: 'Are you sure you want to delete the user?'
+  confirmation_deletion_tag: 'Are you sure you want to delete the tag?'
```

переводы можно сруппировать для понятности

```suggestion
  edit: 'Edit'
  delete: 'Delete'
  add: 'Add'
  remove: 'Remove'
  actions: 'Actions'
  available_tasks: 'Available tasks'
  filter_by_tags: 'Filter by tags'
  reset: 'Reset'
  created_at: 'Created at'
  tags:
    name: 'Tags name'
    create: 'Create tag'
    enter_name: 'Enter name'
    edit: 'Edit tag'
    confirmation_deletion: 'Are you sure you want to delete the tag?'
  users:
    email: 'Email'
    role: 'Role'
    index: 'List of users'
    edit: 'Edit users'
    confirmation_deletion: 'Are you sure you want to delete the user?'
```

---

частично исправлено